src/main/java/com/epam/emotionalhelp/controller/dto/EmotionDto.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.emotionalhelp.controller.dto;
+
+import lombok.Data;
+
+@Data
```

Comment:
Can we consider [@Value](https://projectlombok.org/features/Value) annotation instead?